packages/appium/docs/en/guides/context.md

```diff
@@ -2,4 +2,40 @@
 title: The Context API
 ---
 
-TODO
+A common feature of many app platforms is the ability for developers to embed web content inside of
+the platform-native app frame. This allows developers to leverage web technologies or existing web
+content for some or all of the app functionality. However, the additional complexity of mixing
+"modes" within a single application can make it difficult for automation tools that are designed to
+target the "native" elements and behaviours.
+
+Appium provides a set of APIs for working with different app modes, called "contexts", that Appium
+drivers can implement if they support automation commands in these different modes. There are three
+basic commands that Appium has added to the W3C WebDriver spec for this purpose:
```

Comment:
it also probably makes sense to mention that drivers could extend this concept via extension commands, like `mobile: getContexts`

Reply:
i kind of wanted to avoid that since this page is just about the API, and then when you read the driver docs you would theoretically know what `mobile: getContexts` is about. as the latter is a non appium standard extension

Reply:
open to other opinions though